### PR TITLE
Add option of reviewerGUID

### LINKED ISSUE #996: PRs shouldn't get auto assigned to MLInfraSwat - instead take a cmd line parameter
These shouldn't be hardcoded and instead should be part of the release pipeline of each team - this way each team can set it to the appropriate GUID and searching through their PRs would be easier as well.

https://github.com/dotnet/roslyn-tools/blob/24bb13d9ebe6e50cd976529c8b7d625954d7bfe2/src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.cs#L27

### PR DESCRIPTION
**Pull request by dmonroym**
You can now specify who you want to default as a reviewer in your PRs. This fixes #996 and should relieve the amount of noise MLInfraSwat was receiving.

### CODE REVIEW

File: src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.cs [hunk 1]
```diff
@@ -434,14 +432,17 @@ public static partial class RoslynInsertionTool
...
-                        // then add the build queuer as a reviewer instead of mlinfraswat.
+                        // If this insertion was queued for PR validation, for a dev branch, for a feature branch,
+                        // or if no default reviewer is specified, then add the build queuer as a reviewer.
                         var isPrValidation = !string.IsNullOrEmpty(GetBuildPRNumber(buildToInsert));
                         var isDevOrFeatureBranch = Options.ComponentBranchName.StartsWith("dev/") || Options.ComponentBranchName.StartsWith("features/");
+                        bool hasReviewer = !string.IsNullOrEmpty(Options.ReviewerGUID);
 
-                        var reviewerId = isPrValidation || isDevOrFeatureBranch
+                        //Easiest way to get the reviewer GUIDs is to create a PR search in AzDo
+                        //You'll get something like https://dev.azure.com/devdiv/DevDiv/_git/VS/pullrequests?_a=active&createdBy=GUID-here
```
Comment (RikkiGibson) [inline src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.cs:442]:
```suggestion
                        // Easiest way to get the reviewer GUIDs is to create a PR search in AzDo
                        // You'll get something like https://dev.azure.com/devdiv/DevDiv/_git/VS/pullrequests?_a=active&createdBy=GUID-here
```

**Review by RikkiGibson — APPROVED** [review]
Looks good. Could you please link an insertion created with this change if possible?

**Comment by dmonroym** [comment]
> Looks good. Could you please link an insertion created with this change if possible?

[This](https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/VS/pullrequest/332560) one contained the GUID for the .NET Project System team instead

**Comment by RikkiGibson** [comment]
Looks great, thanks!

**Pull request merged**